components/application/logout.jsx

```diff
@@ -6,10 +6,11 @@ import styles from './logout.css'
 import { withGlobalStore } from 'store'
 
 const LOGOUT_URL = `https://api.dev.core.ac.uk/logout?continue=${encodeURIComponent(
-  typeof window !== 'undefined' ? `${window.location.origin}/logout.html` : ''
+  typeof window !== 'undefined' ? `${window.location.origin}/?logout` : ''
```

Comment:
Assuming `typeof` always returns a string by definition you [can use `!=` here](https://eslint.org/docs/rules/eqeqeq#smart).

Reply:
I always use strict comparison. Otherwise, my IDE highlights it and I don't want that.

Reply:
If you use JetBrains' IDE, it always highlight too much but it should prefer ESLint config in cases such as this one.

IDE is intelligent by definition but you should keep in mind that it's done by people for people and cannot be more smart than people themselves. So, it's pretty fine that IDE highlights something that should not be highlighted.

Reply:
I don't see anything wrong with using `!==`. The eslint doesn't enforce using `!=` with typeof. Both ways are correct it's just a matter of taste unless